proto/tendermint/liquidity/v1beta1/liquidity.proto

```diff
@@ -74,18 +78,18 @@ message Params {
         }];
 
     // Limit the size of each liquidity pool in the beginning phase of Liquidity Module adoption to minimize risk, 0 means no limit
-    string reserve_coin_limit_amount = 4 [
-        (gogoproto.moretags)   = "yaml:\"reserve_coin_limit_amount\"",
+    string max_reserve_coin_amount = 4 [
+        (gogoproto.moretags)   = "yaml:\"max_reserve_coin_amount\"",
         (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int",
         (gogoproto.nullable)   = false,
         (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
             example: "\"1000000000000\"",
             format: "sdk.Int"
         }];
 
-    // Fee paid for new LiquidityPool creation to prevent spamming
-    repeated cosmos.base.v1beta1.Coin liquidity_pool_creation_fee = 5 [
-        (gogoproto.moretags)   = "yaml:\"liquidity_pool_creation_fee\"",
+    // Fee paid for new Liquidity Pool creation to prevent spamming
```

Comment:
```suggestion
    // Fee to create a Liquidity Pool. 
```

Reply:
do we need to add the "why" we have a fee here? or we can just define the parameter (I think we can just state what the parameter is)